sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Count.scala

```diff
@@ -43,6 +43,9 @@ import org.apache.spark.sql.types._
   since = "1.0.0")
 // scalastyle:on line.size.limit
 case class Count(children: Seq[Expression]) extends DeclarativeAggregate {
+
+  require(children.nonEmpty, s"$prettyName requires at least one argument.")
+
```

Comment:
Can you do it in `checkInputDataTypes`?, e.g.)

```scala
override def checkInputDataTypes(): TypeCheckResult = if (children.length <= 1) {
    TypeCheckResult.TypeCheckFailure("blah blah")
} else {
  TypeCheckResult.TypeCheckSuccess
}
```

Reply:
OK